main/install/configuration.dist.php

```diff
@@ -1088,8 +1088,16 @@
 // - launch composer install to rebuild the autoload.php
 //$_configuration['allow_portfolio_tool'] = false;
 
+// DEPRECATED: Use gradebook_display_extra_stats instead.
```

Comment:
`// DEPRECATED: gradebook_enable_best_score is deprecated. Use gradebook_display_extra_stats instead.`